fronts-client/src/reducers/rootReducer.ts

```diff
@@ -1,40 +1,40 @@
 import { reducer as form, FormStateMap } from 'redux-form';
```

Comment:
Big can of worms here for the next refactor, but it'd be so nice to have consistent names so we don't need to rename on import.

Reply:
Oh yeah - what importing this reducer directly from `redux-form`?